src/sections/navbar.html

```diff
@@ -0,0 +1,38 @@
+<div class="bg-PrimaryColor flex justify-between p-4 fixed w-full top-0 z-10">
+  <div class="flex flex-col text-smallerSize md:text-2xl">
+    <h1>DE</h1>
+    <h1>EN</h1>
+  </div>
+  <div class="text-3xl font-bold relative lg:text-5xl">
+    <img
+      src="/src/sections/img/Vector-5.webp"
+      alt="menu-image"
+      class="md:hidden w-48 h-14"
+    />
+    <p
+      class="absolute left-4 top-2 text-smallSize md:text-5xl md:relative lg:text-5xl"
+    >
+      Baumeister
+    </p>
+  </div>
+  <div class="hidden lg:block">
+    <div class="flex flex-col font-semibold text-2xl">
+      <a href="#about">About</a>
+      <a href="#services">Services</a>
+      <a href="#References">References</a>
+      <a href="#contact">Contact</a>
+    </div>
```

Comment:
Here, the font-weight is regular so rather than using font-semibold you can try font-light and see if it's more suitable